bip-???-cat.mediawiki

```diff
@@ -0,0 +1,94 @@
+<pre>
+  BIP: ???
+  Layer: Consensus (soft fork)
+  Title: OP_CAT
+  Author: Ethan Heilman <ethan.r.heilman@gmail.com>
+          Armin Sabouri <arminsdev@gmail.com>
+  Status: Draft
+  Type: Standards Track
+  Created: 2023-10-21
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-op-cat
+  License: BSD-3-Clause
+</pre>
+
+==Abstract==
+
+This BIP reintroduces OP_CAT in the form of a new tapscript opcode which allows the concatenation of two values on the stack. This opcode would be activated via a soft fork by redefining the opcode OP_SUCCESS126.
```

Comment:
Adding a `(0x7e)` might help be a bit clearer. Libraries in various languages have small variations on OP code naming.

Reply:
Thanks, updated bip